packages/@divvi/mobile/src/components/ReviewTransaction.tsx

```diff
@@ -216,26 +291,27 @@ export function ReviewTotalValue({
 
   // if there are no fees then just format token amount
   if (!feeTokenInfo || !feeTokenAmount) {
+    // if fiat amount is availabke then show both token and fiat amounts
     if (localAmount) {
       return (
-        <Trans
-          i18nKey={'tokenAndLocalAmountApprox'}
-          tOptions={{
-            tokenAmount: formatValueToDisplay(tokenAmount),
-            localAmount: formatValueToDisplay(localAmount),
-            tokenSymbol: tokenInfo.symbol,
-            localCurrencySymbol,
-          }}
+        <ReviewDetailsItemTokenValue
+          approx={approx}
+          tokenAmount={tokenAmount}
+          localAmount={localAmount}
+          tokenInfo={tokenInfo}
+          localCurrencySymbol={localCurrencySymbol}
         >
           <Text style={styles.totalPlusFeesLocalAmount} />
-        </Trans>
+        </ReviewDetailsItemTokenValue>
       )
     }
 
-    return t('tokenAmountApprox', {
+    // otherwise only show token amount
+    const displayTokenAmount = t('tokenAmount', {
```

Comment:
```suggestion
    const displayTokenAmount = t(withApprox ? 'tokenAmountApprox' : 'tokenAmount', {
```

why not preserve the flexibility to change how we display the approx across all similar places in this component? (i don't feel strongly about this, but if this flexibility wasn't considered advantageous, why did we bother introducing keys like `tokenAmountApprox` in the first place?)

Reply:
1. RE the suggestion: I do remember us discussing trying to avoid conditional usage of translation keys for the future purposes of potential static analysis of translation keys? 
It will also prevent us from having type-safety for the translation keys intellisense.

3. Ah, okay, that's my bad: I understood [this comment](https://github.com/divvi-xyz/divvi-mobile/pull/100#discussion_r1971263993) as turning the adding of the approx as a prop in translation files (cause we had it like that on `TokenDisplay`). Should've clarified this. Will change this, but will probably change to this to still keep the ability to implement things necessary for the point 1:
```typescript
return withApprox ? t('localAmountApprox', {...}) : t('localAmount', {...})
```

Reply:
@kathaypacific I've changed this back and also found a way to use [nested interpolation](https://www.i18next.com/translation-function/nesting) inside translation files so they are not as repetitive anymore!